From 26574f878072d60a07a3d6ecbc81c4a592de5034 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@freescale.com>
Date: Fri, 27 Nov 2015 15:09:30 +0800
Subject: [PATCH 0686/5242] MLK-11914-2 video: mipi_dsi: add the missing error
 code check

commit  c9560581d27b3513b5923c5eefb8f81c0d8643da from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the missing error code check just as the others which
have the check.

Signed-off-by: Fancy Fang <chen.fang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c b/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
index 8c0cf4f..285f7e7 100644
--- a/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
+++ b/drivers/video/fbdev/mxc/mxcfb_hx8369_wvga.c
@@ -385,6 +385,7 @@ int mipid_hx8369_lcd_setup(struct mipi_dsi_info *mipi_dsi)
 
 static int mipid_bl_update_status(struct backlight_device *bl)
 {
+	int err;
 	u32 buf;
 	int brightness = bl->props.brightness;
 	struct mipi_dsi_info *mipi_dsi = bl_get_data(bl);
@@ -395,8 +396,9 @@ static int mipid_bl_update_status(struct backlight_device *bl)
 
 	buf = HX8369_CMD_WRT_DISP_BRIGHT |
 			((brightness & HX8369BL_MAX_BRIGHT) << 8);
-	mipi_dsi_pkt_write(mipi_dsi, MIPI_DSI_GENERIC_SHORT_WRITE_2_PARAM,
+	err = mipi_dsi_pkt_write(mipi_dsi, MIPI_DSI_GENERIC_SHORT_WRITE_2_PARAM,
 		&buf, 0);
+	CHECK_RETCODE(err);
 
 	hx8369bl_brightness = brightness & HX8369BL_MAX_BRIGHT;
 
-- 
1.7.9.5

